src/libraries/Microsoft.PowerFx.Connectors/OpenApiExtensions.cs

```diff
@@ -814,9 +822,9 @@ public static HttpMethod ToHttpMethod(this OperationType key)
             };
         }
 
-        public static FormulaType GetReturnType(this OpenApiOperation openApiOperation, ConnectorCompatibility compatibility)
+        public static FormulaType GetReturnType(this OpenApiOperation openApiOperation, ConnectorSettings settings)
```

Comment:
>ConnectorSettings [](http://example.com/codeflow?start=88&length=17)

this is flagging as a public API break. Is this ok?

Reply:
I'll add this one for backward compatibility

        [Obsolete("Use a ConnectorSettings parameter instead")]
        public static FormulaType GetReturnType(this OpenApiOperation openApiOperation, ConnectorCompatibility compatibility)